cmake/Modules/FindAtlas.cmake

```diff
@@ -7,7 +7,7 @@
 #  Atlas_FOUND
 #  Atlas_INCLUDE_DIRS
 #  Atlas_LIBRARIES
-#  Atlas_LIBRARYRARY_DIRS
+#  Atlas_LIBRARY_DIRS
```

Comment:
This probably comes from an upstream somewhere (I may even have a thing for that somewhere...)... happy to drop